### PR TITLE
[DOI-1020] New endpoints for update user and customer

### PR DESCRIPTION
**Pull request by blancfabian**
New endpoints for update user and customer

### CODE REVIEW

File: doppler-beplic/Models/DTO/CustomerUpdateDTO.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+namespace DopplerBeplic.Models.DTO
+{
+    public class CustomerUpdateDTO
```
Comment (fgchaio) [inline doppler-beplic/Models/DTO/CustomerUpdateDTO.cs:3]: why customer? Isn't it User and Company?

**Review by fgchaio — APPROVED** [review]
👍 
I would replace the "customer" naming for "company".
What do you think?

**Comment by fgchaio** [comment]
You are missing the endpoints declaration in `program.cs`

**Comment by RodrigoPereyraDiaz** [comment]
I'm out of context, but looks a good work.

I suggest trying to apply [TTD](https://es.wikipedia.org/wiki/Desarrollo_guiado_por_pruebas) or at less add some tests to maintain the code coverage percentage

**Pull request merged**